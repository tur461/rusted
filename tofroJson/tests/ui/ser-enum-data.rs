use miniserde::Serialize;

#[derive(Serialize)]
enum Enum {
    Variant(i32)
}

fn main() {}
