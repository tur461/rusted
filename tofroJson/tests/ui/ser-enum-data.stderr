error: Invalid variant: only simple enum variants without fields are supported
 --> tests/ui/ser-enum-data.rs:5:5
  |
5 |     Variant(i32)
  |     ^^^^^^^^^^^^
